common/changes/sparo/chore-update-sparo-profile-template_2024-03-14-23-58.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "sparo",
+      "comment": "Add more comments in sparo profile template",
+      "type": "none"
+    }
+  ],
+  "packageName": "sparo"
+}
```

Comment:
Don't forget to update the duplicated strings in these files:

https://github.com/tiktok/sparo/blob/main/apps/website/docs/pages/configs/profile_json.md

https://github.com/tiktok/sparo/blob/main/apps/website/static/schemas/sparo-profile.schema.json